src/pudl/package_data/settings/etl_fast.yml

```diff
@@ -11,17 +11,17 @@ ferc_to_sqlite_settings:
   ferc2_dbf_to_sqlite_settings:
     years: [2019, 2020]
   ferc2_xbrl_to_sqlite_settings:
-    years: [2021, 2023]
+    years: [2021, 2024]
   ferc6_dbf_to_sqlite_settings:
     years: [2019, 2020]
   ferc6_xbrl_to_sqlite_settings:
-    years: [2021, 2023]
+    years: [2021, 2024]
   ferc60_dbf_to_sqlite_settings:
     years: [2019, 2020]
   ferc60_xbrl_to_sqlite_settings:
-    years: [2021, 2023]
+    years: [2021, 2024]
   ferc714_xbrl_to_sqlite_settings:
-    years: [2021, 2023]
+    years: [2021, 2023, 2024]
```

Comment:
I added 2024 and didn't remove 2023 here because we still need the 2023 data to be available, until we've updated the fast PUDL ETL to run on 2024 data.